website/content/docs/k8s/annotations-and-labels.mdx

```diff
@@ -74,83 +74,96 @@ The following Kubernetes resource annotations could be used on a pod to control
 - `consul.hashicorp.com/connect-service-upstreams` - The list of upstream
   services that this pod needs to connect to via the service mesh along with a static
   local port to listen for those connections. When transparent proxy is enabled,
-  this annotation is optional. There are a few formats this annotation can take:
+  this annotation is optional. This annotation can be either _unlabeled_ or _labeled_. Unlabeled annotations use component names, and the component types are inferred through the annotation's syntax. Labeled annotations explicitly associate component names and types to form Consul DNS addresses. To configure upstreams for [cluster peering](/consul/docs/k8s/connect/cluster-peering/tech-specs), use the labeled format.
 
-    - Unlabeled:
+    - **Unlabeled**:
        Use the unlabeled annotation format to specify a service name, Consul Enterprise namespaces and partitions, and
-       datacenters. To use [cluster peering](/consul/docs/k8s/connect/cluster-peering/tech-specs) with upstreams, use the following
-       labeled format.
-      - Service name: Place the service name at the beginning of the annotation to specify the upstream service. You can
-      also append the datacenter where the service is deployed (optional).
+       datacenters. 
+
+      - Service name: Place the service name at the beginning of the annotation to specify the upstream service. You also have the option to append the datacenter where the service is deployed.
+   
         ```yaml
         annotations:
           "consul.hashicorp.com/connect-service-upstreams":"[service-name]:[port]:[optional datacenter]"
         ```
 
       - Namespace (requires Consul Enterprise 1.7+): Upstream services may be running in a different namespace. Place
-      the upstream namespace after the service name. For additional details about configuring the injector, refer to
-
-      [Consul Enterprise Namespaces](#consul-enterprise-namespaces) .
+      the upstream namespace after the service name. For additional details about configuring the injector, refer to [Consul Enterprise namespaces](#consul-enterprise-namespaces) .
+        
         ```yaml
         annotations:
           "consul.hashicorp.com/connect-service-upstreams":"[service-name].[service-namespace]:[port]:[optional datacenter]"
         ```
+
         If the namespace is not specified, the annotation defaults to the namespace of the source service.
-        If you are not using Consul Enterprise 1.7+, Consul interprets the value placed in the namespace position as part of the service name.
+        If your version of Consul Enterprise in lower than v1.7, Consul interprets the value placed in the namespace position as part of the service name.
```

Comment:
```suggestion
        Consul Enterprise v1.7 and older interprets the value placed in the namespace position as part of the service name.
```
Did we decide on referring to versions in temporal or cardinal order?

Reply:
Tangential: Do we still need explanations like this now that we have versioned docs? Or should we remove things that talk about differences in behavior in fairly old versions?

It's not something we have to decide for this PR. It probably makes sense to make a decision and then apply across the docs surface if needed rather than piecemeal in each PR.